src/modules/home/heroSlider.tsx

```diff
@@ -8,31 +7,27 @@ import {
   A11y,
   Autoplay,
 } from "swiper/modules";
-import Image from "next/image";
+import { CldImage } from "next-cloudinary";
 
 const heroImages = [
   {
     id: 1,
-    imageSrc:
-      "https://raw.githubusercontent.com/xplorer-io/website-images-cdn/refs/heads/main/xplorers-website-images/homepage/hero-image-1.webp",
+    imageSrc: "homepage/aayttz2ixe1qrx9cjpaa",
     imageAlt: "group image 1",
   },
   {
     id: 2,
-    imageSrc:
-      "https://raw.githubusercontent.com/xplorer-io/website-images-cdn/refs/heads/main/xplorers-website-images/homepage/hero-image-2.webp",
+    imageSrc: "Collected_Images_2984_nguvyu",
```

Comment:
we can probably put some better naming, but we can always do it later.
good for now!